javascript-timers/interval.js

```diff
@@ -0,0 +1,17 @@
+const h1 = document.querySelector('h1');
+
+let myTimer = setInterval(handleSetInterval, 1000);
+function handleSetInterval() {
+  console.log('hi');
```

Comment:
If they're not part of the requirements, please remove `log`s to the `console` after you're done using them to test things.